### PR TITLE
[Hub Generated] Publish private branch 'avjan/new-api-version-for-vasetteings'

### PR DESCRIPTION
**Pull request by AvishalomJ**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/published%2FNomigori96MS%2Fazure-rest-api-specs-pr%2Favjan%2Fnew-api-version-for-vasetteings...main/).
### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/overview_breakingchanges).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](http://aka.ms/aprwiki))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by AvishalomJ** [comment]
The PR is created based on the updates in the private branch. The updates in the PR has already been reviewed and approved with this PR [Azure/azure-rest-api-specs-pr/12575](https://github.com/Azure/azure-rest-api-specs-pr/pull/12575)

**Comment by mentat9** [comment]
@AvishalomJ - Please carefully and completely fill out the submission form. Your PR cannot be routed correctly and reviewed without that information.

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    "ServerVulnerabilityAssessmentsSetting": {
+      "type": "object",
+      "description": "A base vulnerability assessments setting on servers in the defined scope.",
+      "properties": {
+        "kind": {
+          "description": "The kind of the server vulnerability assessments setting.",
+          "$ref": "#/definitions/ServerVulnerabilityAssessmentsSettingKind"
+        },
+        "systemData": {
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:222]: "systemData" already defined in "../../../common/v1/types.json#/definitions/Resource"

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSettingsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:67]: It looks the same, so I think it's possible. I'll update it.

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:149]: It looks the same, so I think it's possible. I'll update it.

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+          "200": {
+            "description": "OK - Setting was deleted"
+          },
+          "204": {
+            "description": "No Content - Setting does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:186]: It looks the same, so I think it's possible. I'll update it.

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Get a list of all the server vulnerability assessments settings over a subscription level scope",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:51]: I don't see the v5 folder under "../../../common/". There is only a v1 folder there.

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          },
+          {
+            "$ref": "#/parameters/SettingBody"
+          }
+        ],
+        "responses": {
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:139]: Done

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json (hunk 1 above)
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:228]: I can't find systemData under the resource definition, did I miss something?

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (ms-zhenhua) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:51]: "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Get a server vulnerability assessments setting of the requested kind, that is set on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:90]:
```suggestion
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:93]:
```suggestion
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Create or update a server vulnerability assessments setting of the requested kind on the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:127]:
```suggestion
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:130]:
```suggestion
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 11]
```diff
@@ -0,0 +1,334 @@
...
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Delete the server vulnerability assessments setting of the requested kind from the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:173]:
```suggestion
           "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 12]
```diff
@@ -0,0 +1,334 @@
...
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:176]:
```suggestion
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"

```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json (hunk 1 above)
Comment (ms-zhenhua) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:228]: "../../../../../common-types/resource-management/v5/types.json#/definitions/Resource"

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 13]
```diff
@@ -0,0 +1,334 @@
...
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/systemData"
+        }
+      },
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
```
Comment (ms-zhenhua) [inline specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:238]:
```suggestion
          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/Resource"
```

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json [hunk 14]
```diff
@@ -0,0 +1,334 @@
...
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Get a list of all the server vulnerability assessments settings over a subscription level scope",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
```
[the remark this answers is not in the document]
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:54]: Done

File: specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json (hunk 13 above)
Comment (AvishalomJ) [reply specification/security/resource-manager/Microsoft.Security/stable/2023-05-01/ServerVulnerabilityAssessmentsSettings.json:238]: Done and removed systemData property

**Review by chunyu3 — APPROVED** [review]

**Pull request merged**